core/src/main/scala/kafka/tools/ProducerPerformance.scala

```diff
@@ -37,7 +37,7 @@ import org.apache.log4j.Logger
  *
  * @note This class will be replaced by org.apache.kafka.tools.ProducerPerformance after the old producer client is removed
  */
-@deprecated
+@deprecated("Replaced by org.apache.kafka.tools.ProducerPerformance", "12-09-2015")
```

Comment:
I can change this to be a version. I put a date because it wan't actually marked for 0.9.0.0, and when developing patches its hard to know exactly what version a change will make it into.